tests/fpsemi_intf.py

```diff
@@ -51,8 +51,12 @@ def check_validation(self, t):
             + e
         )
 
-    with self.assertRaises(TypeError):
-        x.validate_word([0, 1, 2])
+    if cppyy_version() <= "1.7.0":
```

Comment:
I think you need to use `compare_version_numbers` for this, rather than using string comparison.